### DISCUSSION TITLE
.js  refrence

### DISCUSSION THREAD

**Discussion by arash-orooji** [google/model-viewer#1940] [Q&A]
Hi fiend's

Whats is the reasons behind .js references ?

for example:
import ModelViewerElementBase, {$hasTransitioned, $needsRender, $onModelLoad, $renderer, $scene, $tick, $updateSource} from '../model-viewer-base.js';

Thanks for your support.

**Comment by elalish** [comment]
The TS compiler creates corresponding .js files for each .ts file, then they import each other. Do you want to become a contributor? These are just internal details if you prefer to be a user.
